sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -0,0 +1,341 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Model factory that enables mocking for the Text Analytics library.
+    /// </summary>
+    public static class TextAnalyticsModelFactory
+    {
+        #region Common
+        /// <summary>
+        /// Initializes a new instance of <see cref="TextAnalytics.TextDocumentStatistics"/> for mocking purposes.
+        /// </summary>
+        /// <param name="grahemeCount">Sets the <see cref="TextDocumentStatistics.GraphemeCount"/> property.</param>
```

Comment:
Spelling of graphemeCount

Reply:
Thank you Anne!! PR https://github.com/Azure/azure-sdk-for-net/pull/11077